README.md

```diff
@@ -220,6 +220,40 @@ language plpgsql;
 -- +goose StatementEnd
 ```
 
+## Embedded sql migrations
```

Comment:
Can you just make a note of the exceptions, `fix` and `create` .. something along the lines of:

"Operations such as `fix` and `create` will continue to operate on the underlying filesystem even when using embedded files. This is expected since these are modifying operations on the directory itself"